### ISSUE TITLE
[CS1.6,CS/CZ] Dead CTs can see bomb carrier in scoreboard

### ISSUE THREAD

**Issue by ConnorMcLeod** [ValveSoftware/halflife#1938]
CT shouldn't be able to see who is carrying C4.
Now that any team is playing with teamspeak or any similar vocal, dead CT can tell their teamates if C4 is carried or not.

**Comment by SamVanheer** [comment]
@mikela-valve Did the recent changes to the scoreboard fix this?

**Comment by mikela-valve** [comment]
@SamVanheer No, that didn't change. This is another issue where this looks to be intentional behaviour so I'm not sure it should be changed unless there is overwhelming consensus. 

There are explicit checks for the current player being dead for showing whether another player has the bomb or is a VIP as well as showing them to spectators, so it seems to be that that was at least not an unintended change. I could add a server cvar for not sending bomb or VIP flags to players of the opposing team if set but I wouldn't want to just make a sweeping change to it unless it seems obvious to.

**Comment by perforatorRU** [comment]
Its rly not a bug? ;D

**Comment by mikela-valve** [comment]
It's not apparent that it was unintentional. Competitive play may not have been considered when that was implemented. I'll mark this as future release for now and come back to it later. I think implementing a server cvar to control this is a good idea in any case, the only open question is whether showing bomb/VIP when dead should default to on or off.

**Comment by RauliTop** [comment]
> I could add a server cvar for not sending bomb or VIP flags to players of the opposing team if set but I wouldn't want to just make a sweeping change to it unless it seems obvious to.

Simple and best option.

Will be useful for competitive servers

**Comment by Splatt581** [comment]
I agree, it would be good to add the control cvar `mp_scoreboard_showflags` with values of `0`-`5` like the cvars `mp_scoreboard_showhealth` / `mp_scoreboard_showmoney`. By default, I think we need to follow similar cvars and send flags only to team members (`3` value).

**Comment by Maxi605** [comment]
> It's not apparent that it was unintentional. Competitive play may not have been considered when that was implemented. I'll mark this as future release for now and come back to it later. I think implementing a server cvar to control this is a good idea in any case, the only open question is whether showing bomb/VIP when dead should default to on or off.

It should be off by default. Considering the way the game is currently played, seems like the best option.
